WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java

```diff
@@ -384,4 +394,15 @@ private void showAlertForFailureReason(int reasonResId) {
         builder.setPositiveButton(R.string.ok, (dialog, which) -> dialog.dismiss());
         builder.show();
     }
+
+    private void showAlertForFacebookPageFailureReason() {
```

Comment:
This dialog and the one below it isn't adhering to the new material theme. Should I create an issue to track this @khaykov or it's something you plan to work on? Let me know the best course of action. Thanks 🙏

Reply:
No worries! I'll make a mental note to update it :)

Reply:
Awesome! Also if you want a Facebook account to test with I could always set up a temp password for the one I used because I know it can be a bit annoying to do this stuff with social networks. Let me know